pkg/registry/clusterrole/strategy.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package clusterrole
+
+import (
+	"fmt"
+
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/apis/rbac"
+	"k8s.io/kubernetes/pkg/apis/rbac/validation"
+	"k8s.io/kubernetes/pkg/fields"
+	"k8s.io/kubernetes/pkg/labels"
+	"k8s.io/kubernetes/pkg/registry/generic"
+	"k8s.io/kubernetes/pkg/runtime"
+	"k8s.io/kubernetes/pkg/util/validation/field"
+)
+
+// clusterRoleStrategy implements behavior for ClusterRoles
+type clusterRoleStrategy struct {
+	runtime.ObjectTyper
+	api.NameGenerator
+}
+
+// clusterRoleStrategy is the default logic that applies when creating and updating
+// ClusterRole objects.
+var Strategy = clusterRoleStrategy{api.Scheme, api.SimpleNameGenerator}
+
+// NamespaceScoped is true for ClusterRoles.
+func (clusterRoleStrategy) NamespaceScoped() bool {
+	return false
```

Comment:
I won't block on this issue, but please fix later if @kubernetes/sig-api-machinery suggests something else.